heron/stmgr/tests/cpp/server/stmgr_unittest.cpp

```diff
@@ -228,7 +228,7 @@ void StartDummyStMgr(EventLoopImpl*& ss, DummyStMgr*& mgr, std::thread*& stmgr_t
 
   mgr = new DummyStMgr(ss, options, stmgr_id, LOCALHOST, stmgr_port, LOCALHOST, tmaster_port,
                        shell_port, instances);
-  mgr->Start();
+  EXPECT_EQ(0, mgr->Start()) << stmgr_port;
```

Comment:
What does << stmgr_port means? Does it just print stmgr_port when the test fails or succeeds?